homeassistant/helpers/schema_config_entry_flow.py

```diff
@@ -52,7 +52,7 @@ class SchemaFlowFormStep(SchemaFlowStep):
     - The `validate_user_input` should raise `SchemaFlowError` is user input is invalid.
     """
 
-    next_step: Callable[[dict[str, Any]], str] | str | None = None
+    next_step: Callable[[dict[str, Any]], str | None] | str | None = None
```

Comment:
Re-added to make it possible to distinguish between an import of data by frontend and a user flow.